pkg/project/auth/watch.go

```diff
@@ -147,9 +147,14 @@ func (w *userProjectWatcher) GroupMembershipChanged(namespaceName string, users,
 			return
 		}
 
+		projectObj, err := projectutil.ConvertNamespaceExternal(namespace)
```

Comment:
this change looks wrong.  Why did you change the type?

Reply:
> this change looks wrong. Why did you change the type?

Ok I see.  See my comments on that method. No error, just build the right value.

Reply:
Thanks, addressed this using your suggestion